lib/ProgressPlugin.js

```diff
@@ -30,8 +31,8 @@ const createDefaultHandler = profile => {
 			}
 			for (let detail of details) {
 				if (!detail) continue;
-				if (detail.length > 40) {
-					detail = `...${detail.substr(detail.length - 39)}`;
+				if (msg.length + detail.length > maxLength) {
+					detail = `…${detail.substr(-(maxLength - msg.length - 2))}`;
```

Comment:
```suggestion
					detail = `...${detail.substr(-(maxLength - msg.length - 2))}`;
```

We tried using this before, but it resulted in problems on some platforms.

Reply:
thought that could be a problem